projects/components/src/datagrid/datagrid.component.ts

```diff
@@ -131,7 +131,7 @@ export class DatagridComponent<R> implements OnInit {
      * Fired whenever the selection changes. The event data is array of rows selected. The array will contain only one
      * element in case of single selection
```

Comment:
Can we add a method to get the selection without having to do HTML binding? Just getting a reference to the vcd grid? Otherwise we need to add state to the parent for a variable holding the selection. The method lets us grab it just when needed, we don't always care about all the changes.